### DISCUSSION TITLE
Unable to interpret value as number when they start with "$"

### DISCUSSION THREAD

**Discussion by valantur** [grafana/grafana-csv-datasource#185] [Q&A]
Hi and thanks for this plugin! the main reason why I use Grafana! 
The issue I'm having is that I can't figure out how to tell Grafana or this plug-in that the values in certain fields are numbers, even though they begin with $.

For example:
date,balance
01-01-2021,$120

Without any transformations, the plug-in thinks both fields are strings. I am able to transform "date" to time and that works fine. However, when I try to convert "$120" to numeric, I lose the value and nothing is displayed. Even weirder, some times the values are just blank, some others are "0".

Is there any way to make this work?

Thanks!

**Comment by marcusolsson** [comment]
This is an interesting use case! I don't think there's a way to solve this at the moment, unless you somehow preprocess the data to remove the dollar sign.

Would be interesting to add support for this, but not sure what the feature would be. Possibly adding basic text transformation to trim a suffix/prefix before parsing the value.
